lib/mix/test/fixtures/test_stale/mix.exs

```diff
@@ -0,0 +1,9 @@
+defmodule TestStale.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :test_stale,
+     version: "0.0.1",
+     test_pattern: "*_test_stale.exs"]
```

Comment:
This is required here due to how the *_test targets are constructed in the Makefile.